src/js/providers/tracks-mixin.js

```diff
@@ -517,15 +517,35 @@ function addVTTCuesToTrack(track, vttCues) {
 // ////////////////////
 
 function _addCueToTrack(renderNatively, track, vttCue) {
-    if (!(Browser.ie && renderNatively) || !window.TextTrackCue) {
-        track.addCue(vttCue);
-        return;
+    let cue = vttCue;
+    if (Browser.ie && renderNatively && window.TextTrackCue) {
+        // There's no support for the VTTCue interface in IE/Edge.
+        // We need to convert VTTCue to TextTrackCue before adding them to the TextTrack
+        // This unfortunately removes positioning properties from the cues
+        cue = new window.TextTrackCue(vttCue.startTime, vttCue.endTime, vttCue.text);
+    }
+
+    // IE/Edge will throw an exception if cues are not inserted in time order: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/13183203/
+    if (Browser.ie) {
+        insertCueInOrder(track, cue);
+    } else {
+        track.addCue(cue);
+    }
+}
+
+function insertCueInOrder(track, vttCue) {
+    const temp = [];
+    const cues = track.cues;
```

Comment:
`track.cues` will be `null` if `track.mode === 'disabled'`. If you need to remove cues, to be safe, you'll need to ensure the mode is either 'hidden' or 'showing' before iterating through the cues. Take a look at [this example](https://github.com/jwplayer/jwplayer/blob/7054d14f331ca7168e5d807c5e6aa7ca14cd5cde/src/js/providers/tracks-mixin.js#L107). Oddly enough, `track.addCue` works even if the track is disabled, which is why there wasn't a need to check the mode before.